packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -21,7 +21,30 @@ const agent = {
 }
 
 async function worker([url, options]) {
-  return got(url, { agent, ...options })
+  return got(url, {
+    agent,
+    ...options,
+    cache: {
+      get: key => options.cache.get(key),
+      set: (key, value) => {
+        const parsed = JSON.parse(value)
+        // Drupal users often set very long max-age as they want to maximize
+        // the max-age of their varnish cache. This would make gatsby-source-drupal
+        // cache API calls for hours or days and people would wonder why
+        // their content doesn't update. We'll change `cache-control` so every request
+        // must revalidate.
+        try {
+          delete parsed.value.cachePolicy.rescc[`max-age`]
+          delete parsed.value.cachePolicy.resh[`cache-control`]
+          parsed.value.cachePolicy.rescc[`must-revalidate`] = true
+          parsed.value.cachePolicy.rescc = {}
```

Comment:
Will this not wipe the `must-revalidate`?

Reply:
oh right — I added that for testing but don't want it. Removing it